Formula/irssi.rb

```diff
@@ -6,11 +6,6 @@ class Irssi < Formula
   license "GPL-2.0-or-later"
   revision 1
 
```

Comment:
```suggestion

  # This formula uses a file from a GitHub release, so we check the latest
  # release version instead of Git tags.
  livecheck do
    url :stable
    strategy :github_latest
  end
```

Since the `stable` URL is a GitHub release download, we need to check releases instead of tags. In general, sometimes there's a delay between when a tag is created and the related release is created (I've seen some projects where this can take days). If we check Git tags, then livecheck can report a new version before the release tarball is available to download and that poses a problem.